projects/framework-premiere/source/ftrack_framework_premiere/__init__.py

```diff
@@ -0,0 +1,281 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+import logging
+import time
+import sys
+import os
+import traceback
+import platform
+import subprocess
+import re
+from functools import partial
+
+try:
+    from PySide6 import QtWidgets, QtCore
+except ImportError:
+    from PySide2 import QtWidgets, QtCore
+
+
+import ftrack_api
+
+from ftrack_constants import framework as constants
+
+from ftrack_utils.extensions.environment import (
+    get_extensions_path_from_environment,
+)
+from ftrack_utils.rpc import JavascriptRPC
+from ftrack_utils.process import MonitorProcess, terminate_current_process
+from ftrack_utils.usage import set_usage_tracker, UsageTracker
+from ftrack_qt.utils.decorators import invoke_in_qt_main_thread
+
+
+from ftrack_framework_core.host import Host
+from ftrack_framework_core.event import EventManager
+from ftrack_framework_core.client import Client
+from ftrack_framework_core.configure_logging import configure_logging
+from ftrack_framework_core import registry
+
+# Evaluate version and log package version
+try:
+    from ftrack_utils.version import get_version
+
+    __version__ = get_version(
+        os.path.basename(os.path.dirname(__file__)),
+        os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
+    )
+except Exception:
+    __version__ = '0.0.0'
+
+configure_logging(
+    'ftrack_framework_premiere',
+    extra_modules=['ftrack_qt'],
+    propagate=False,
+)
+
+logger = logging.getLogger(__name__)
+logger.debug('v{}'.format(__version__))
+
+premiere_rpc_connection = None
+
+# Create Qt application
+app = QtWidgets.QApplication.instance()
+
+if not app:
+    app = QtWidgets.QApplication(sys.argv)
+    app.setAttribute(QtCore.Qt.AA_PluginApplication)
+
+remote_session = None
+
+process_monitor = None
```

Comment:
Can this be aligned with PS? 
photoshop_rpc_connection = object()
remote_session = object()
process_monitor = object()